docs: fly cli integraton (#79)
depot/docs: content/integrations/fly.mdx

Once you have a Fly.io account, you can create and deploy a new app using the Fly CLI. You can install the Fly CLI using the methods described in the [Fly.io documentation](https://fly.io/docs/flyctl/install/).

> You have two options for intefrating Depot with Fly.io, you may build from your Depot account and push to Fly.io, or you can use the `--depot` flag with the Fly deploy command to use Depot as the builder on Fly.

Reviewer: ```suggestion
You have two options for integrating Depot with Fly.io, you may build the image with `depot build` using your Depot account and push it to Fly.io, or you can use the `--depot` flag with the `flyctl deploy` command to use Depot as the builder on Fly.
```

## Using Depot with Fly.io

### Fly CLI

> When using Depot as the builder for your Fly.io apps, you will not need to connect a Depot account. Simply specify Depot as the builder with the `--depot` when deploying and automatically take advantage of Depot's accelerated builds.

Reviewer: ```suggestion
When using Depot as the builder for your Fly.io apps, you will not need to connect a Depot account. Simply specify Depot as the builder with the `--depot` flag when deploying and automatically take advantage of Depot's accelerated builds.
```
